src/scripts/main.js

```diff
@@ -1,3 +1,48 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
```

Comment:
Ensure that the '.logo' element exists in the DOM before attaching an event listener. If it doesn't exist, `document.querySelector` will return `null`, and attempting to add an event listener will throw an error.